src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java

```diff
@@ -103,6 +104,20 @@ public void createCase(@NotNull String caseType, @NotNull String dateReceived, @
         }
     }
 
+    public void migrateCase(@NotNull String caseType, @NotNull String dateReceived, @NotNull String fromCaseUUID) {
+        UUID caseUuid = UUID.fromString(fromCaseUUID);
+        GetCaseworkCaseDataResponse caseData = caseworkClient.getCase(caseUuid);
```

Comment:
I'm thinking on reflection that the logic to create the new Case of the new case type, might be better suited to take place in the `Casework Service` making this a very quick and easy pass through method. That way you could avoid making several Rest calls for the process. 

Just one migrate call, passing along the caseUUID, stageUUID, and the new casetype. What do you think?

We could also eliminate all of this data mapping here and in the `WorkflowService` class